pages/standards-guidelines/wcag/non-web-ict.md

```diff
@@ -44,24 +44,64 @@ Quick link:
+### Goals
+The objective of the WCAG2ICT Task Force is to develop documentation describing how WCAG 2.x and its principles, guidelines, and success criteria could apply to non-Web Information and Communications Technologies (ICT).
+
+### Intent and Usage of WCAG2ICT
+
+WCAG2ICT, like any other  [W3C Group Note](https://www.w3.org/policies/process/#Note), is informative — it is not normative and does not set requirements. It provides guidance on the application of WCAG 2 principles, guidelines, and success criteria to non-web documents and software. The WCAG2ICT Group Note was developed to provide an authoritative interpretation of how WCAG 2 can apply in different contexts.
+
+The main focus of the WCAG2ICT Group Note is to provide verbiage replacements for web-based language in the WCAG success criteria and definitions as well as provide additional insights into where WCAG criteria may make assumptions of the presence and functionality of a user agent and assistive technology. This is because not all software has a user agent or platform software with accessibility APIs and services, nor do all technologies have or support assistive technology that acts on programmatic information. 
+
+While the WCAG2ICT Group Note provides guidance on applying WCAG in non-web contexts, at this time it may not provide sufficient clarity for some success criteria on how they can be applied to electronic documents whose source is driven by non-web markup technologies when they are delivered via a website. Likewise, there may not be sufficient clarity for mobile applications.  
 
-*Scope*: WCAG2ICT does not include techniques for how to meet WCAG success criteria in specific technologies. More information is in the [Excluded from Scope section](https://www.w3.org/TR/wcag2ict-22/#excluded-from-scope).
 
 ### Versions
-* [Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-22/) was published 8 October 2024 and covers WCAG 2.0, 2.1, and 2.2
+* [Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-22/) was published 15 November 2024 and covers WCAG 2.0, 2.1, and 2.2
 * older version: [Guidance on Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-20/) was published 5 September 2013 and covers WCAG 2.0
 * The URI [https://www.w3.org/TR/wcag2ict/](https://www.w3.org/TR/wcag2ict/) points to the latest completed version
 
 ## Who WCAG2ICT is for {#for}
 
-WCAG2ICT is for ICT managers, ICT developers, policy makers, and others wanting to understand how WCAG 2 can be applied to non-web documents and software.
+The WCAG2ICT Group Note is for standards developers, ICT managers, ICT developers, policy makers, and any others wanting to understand how WCAG 2 can be applied to non-web documents and software.
```

Comment:
W3C has traditionally avoided "policy makers" as an audience. Could we do with "regulators" as a broader term? Applies to subsequent appearances of the word.

```suggestion
The WCAG2ICT Group Note is for standards developers, ICT managers, ICT developers, regulators, and any others wanting to understand how WCAG 2 can be applied to non-web documents and software.
```

Reply:
1) Task Force Reviewed. We do not want to accept change. Policy makers is more accurate.  Policy makers include people inside a company or organization.

Reply:
@ChrisLoiselle 

Thanks. I understand TF rationale. I'd like to leave this open in case this serves as guidance for other WAI Staff that will review this.